tests/integration/instance-initializers/axe-component-test.js

```diff
@@ -1,14 +1,14 @@
 /* global sinon, axe */
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
 import Component from '@ember/component';
-
 import Ember from 'ember';
 import { initialize } from 'dummy/instance-initializers/axe-component';
 import {
   initialize as initializeViolationsHelper
 } from 'dummy/instance-initializers/violations-helper';
-import { moduleForComponent, test } from 'ember-qunit';
-import hbs from 'htmlbars-inline-precompile';
-import wait from 'ember-test-helpers/wait';
 
 const {
   Logger
```

Comment:
We need to remove the use of `Ember.Logger`:

* https://emberjs.github.io/rfcs/0297-deprecate-ember-logger.html
* https://deprecations.emberjs.com/v3.x#toc_ember-console-deprecate-logger

Using `Ember.Logger` will emit warnings/deprecations since Ember 3.2.

---

This can (and probably should) be handled in a separate PR, I just noticed it while reviewing here...

Reply:
https://github.com/ember-a11y/ember-a11y-testing/issues/127